packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,254 @@
+
+> NOTE: THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+- [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+- [ViD Select Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=2597-83231&mode=design)
+
+## Usage
+When to use:
+- To present a pre-defined list of options
+- For filtering or sorting data in tables or lists
+- For selecting settings or preferences
+- For single selection
+
+When not to use:
+- To allow end-users to add custom options
+- When there are fewer than three options
+- For complex and/or hierarchical data
+- For multi selection
+
+### Anatomy
+![ ](spec-images/select-1.png)
+
+| Element | Description |
+| ---------- | ---------- |
+| Label | Text that tells the end-user what to expect in the list of options | 
+| Helper text | Text that provides extra context or instruction about what the end-user should select | 
+| Field button | Click area that opens the menu and displays the current selected option| 
+| Menu | A list of options to choose from displayed as an *open* state | 
+| Menu item | A selection the end-user can make, shown with other options in the menu | 
+
+#### Label
+Selects should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases).
+
+Keep the label text as concise as possible, limited to a single line of text. Use sentence casing.
+
+In a form, use the label to indicate whether the select is optional or required for the end-user to interact with. If most of the components in the form are required, indicate which components are optional. If most of the components in the form are optional, indicate which components are required. Required or optional indication in the label text should always follow the pattern "[Label] (optional)" or "[Label] (required)".
+
+![ ](spec-images/select-2.png)
+
+#### Helper Text
+> NOTE: NEEDS VISUAL DESIGN FOR TYPOGRAPHY
+
+Only include helper text to provide information that aids the end-user in choosing the right selection. Keep the helper text concise as possible, limited to a single line of text. Use sentence casing.
+
+![ ](spec-images/select-3.png)
+
+#### Field Button
+Selects can be empty by default or have a default selection from the menu. Empty selects should include placeholder text that propts the end-user to select an option from the menu. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Ellipses are not needed. Use sentence casing.
+
+![ ](spec-images/select-4.png)
+
+#### Menu & Menu Items
+A list of options to select, displayed as an open state. Order menu items alphabetically by default, unless another ordering will provide value to the end-user. Keep the menu item text concise as possible, limited to a single line of text. Use sentence casing.
```

Comment:
I'm not sure the `Select` should intervene in the ordering of the items in the dropdown. Order should be provided via the DOM order of the options. If the user thinks alphabetical order is appropriate, they should order their options so.